MAUI/TreeMap/selection-and-highlights.md

```diff
@@ -558,4 +558,6 @@ public class PopulationViewModel
 }
 
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+N> This property is applicable only when the `SelectionMode` is set to `Single` or `Multiple.`
```

Comment:
N> This property is applicable only when the `SelectionMode` is set to `Single` or `Multiple`.

Reply:
Updated.